projects/kit/components/input-time/input-time-options.ts

```diff
@@ -1,11 +1,13 @@
 import {InjectionToken} from '@angular/core';
 import {TuiContextWithImplicit, TuiTimeMode} from '@taiga-ui/cdk';
 import {TuiSizeL, TuiSizeS} from '@taiga-ui/core';
+import {TuiTimeFormatParts} from '@taiga-ui/kit/types';
 import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
 
 export interface InputTimeOptions {
     readonly icon: PolymorpheusContent<TuiContextWithImplicit<TuiSizeS | TuiSizeL>>;
     readonly mode: TuiTimeMode;
+    readonly maxValues: Record<TuiTimeFormatParts, number>;
```

Comment:
`Record<TuiTimeFormatParts, number>` can we use more concrete the type?

Reply:
Could you be more specific please? AFAIR, `Record` enforces us to specify all keys

Reply:
sorry, I see that now